.github/workflows/ci.yml

```diff
@@ -244,15 +244,15 @@ jobs:
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 3.1.x
-    - name: Setup .NET SDK v7.0.x
-      if: matrix.dotnet == 'net7.0'
-      uses: actions/setup-dotnet@v3
-      with:
-        dotnet-version: 7.0.x
     - name: Setup .NET SDK v6.0.x
+      if: matrix.dotnet == 'net6.0'
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.x
+    - name: Setup .NET SDK v7.0.x
```

Comment:
Does this need an `if: matrix.dotnet == 'net7.0'`?

Reply:
No because we want to install this SDK in all cases, as it's required to build any target now that we have updated our `global.json`